Packs/FireEyeETP/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### FireEye ETP
+Fixed a bug in the **fireeye-etp-get-alerts** command in which directly accessing fields in the response
+ caused a *KeyError* exception.
```

Comment:
```suggestion
Fixed an issue in the ***fireeye-etp-get-alerts*** command where directly accessing fields in the response would cause a *KeyError* exception.
```